src/services/template/db.test.ts

```diff
@@ -1,6 +1,30 @@
-import { getUserById } from './db';
+const { createTestClient } = require('apollo-server-testing');
 
-test('testing user id', () => {
-  expect(getUserById('1').firstName).toBe('Gil');
-  expect(getUserById('1').lastName).toBe('Amran');
+it('fetches single launch', async () => {
```

Comment:
Be consistent with `it` or `test`. You use `test` for the UI testing so let's just stick to that even though they mean the same thing

Reply:
got it